### PR TITLE
General typos #172

### PR DESCRIPTION
**Pull request by boydell**
Fixes to some minor typos. Also, I personally did not get an invite to the Slack Workspace, so I changed the instructions there, knowing that I don't really know the right way to gain access.

### CODE REVIEW

File: mentoring/how_to_give_great_feedback.md [hunk 1]
```diff
@@ -86,5 +86,5 @@ As you gain experience mentoring, you will get a feel for the different ways stu
 ## Unsure of something? Chat to other mentors?
 
 One of the best things about being an Exercism mentor is the community of other mentors you can learn from.
-If you've got a question about a solution, or just want to bounce ideas with other mentors, jump into our [Slack Workspace](https://exercism-team.slack.com/) (you should receive an invite when you sign up to mentor) and ask away!
+If you've got a question about a solution, or just want to bounce ideas with other mentors, jump into our [Slack Workspace](https://exercism-team.slack.com/) (Send a request to be added to the workspace to hello@exercism.org) and ask away!
```
Comment (ErikSchierboom) [inline mentoring/how_to_give_great_feedback.md]: Could you revert this change? We really don't want people to send us emails.

**Comment by iHiD** [comment]
@boydell Can I ask when you signed up to be a mentor? There was a window of a few hours where yo the invite emails didn't send (I'll resend them now to be sure). But if it was not that window, I'll investigate further.

**Comment by boydell** [comment]
> @boydell Can I ask when you signed up to be a mentor? There was a window of a few hours where yo the invite emails didn't send (I'll resend them now to be sure). But if it was not that window, I'll investigate further.

@iHiD Now I can't remember exactly the process, but it seems like I didn't signup and instead just started doing it. I didn't start until after v3 was launched. The message thread says my first message to someone I was mentoring was 6 days ago.

I just checked my email again, and no invite messages, including my spam folder.

File: mentoring/how_to_give_great_feedback.md (hunk 1 above)
Comment (boydell) [reply mentoring/how_to_give_great_feedback.md]: For sure. :)

**Comment by iHiD** [comment]
Invite sent :)

**Comment by ErikSchierboom** [comment]
@boydell There are now two merge conflicts. Could you resolve them please?

**Comment by boydell** [comment]
> @boydell There are now two merge conflicts. Could you resolve them please?

@ErikSchierboom  Resolved.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Thanks
